src/main/kotlin/no/nav/syfo/melding/cronjob/PaminnelseCronjob.kt

```diff
@@ -0,0 +1,44 @@
+package no.nav.syfo.melding.cronjob
+
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.application.cronjob.Cronjob
+import no.nav.syfo.application.cronjob.CronjobResult
+import no.nav.syfo.melding.kafka.PublishPaminnelseService
+import org.slf4j.LoggerFactory
+
+class PaminnelseCronjob(
+    private val publishPaminnelseService: PublishPaminnelseService,
+    override val intervalDelayMinutes: Long,
+) : Cronjob {
+    override val initialDelayMinutes: Long = 2
+
+    override suspend fun run() {
+        val result = runJob()
+        log.info(
+            "Completed publishing paminnelse processing job with result: {}, {}",
+            StructuredArguments.keyValue("failed", result.failed),
+            StructuredArguments.keyValue("updated", result.updated),
+        )
+    }
+
+    fun runJob(): CronjobResult {
+        val result = CronjobResult()
+        val unpublishedPaminnelser = publishPaminnelseService.getUnpublishedPaminnelser()
```

Comment:
Ref https://github.com/navikt/isbehandlerdialog/pull/63/files#r1210220658 også. Blir litt forvirret av at vi kaller det påminnelser i denne sammenheng. Jeg tenker at når veileder skal trykke på knappen for påminnelse i Modia så oppretter vi en påminnelse-melding og publiserer en påminnelse-dialogmelding til `isdialogmelding`. Mens her skal vi hente utgående meldinger som ikke har fått svar innen fristen og publisere det til en topic. (for å lage oppgaver som skal påminne veileder om å lage påminnelse 🤯 ). Kan vi finne på bedre navn/begreper som gjør dette tydeligere?

Reply:
Jeg er enig, det blir vanskelig å vite hva en påminnelse er; hvis det både er oppgave og brev som sendes 🤷

Reply:
Jepp, jeg lander nok på å bruke `ubesvart` i stedet for `paminnelse` i denne casen 👍🏼 Bra diskusjon!